reference/extensions/profile_plugin.rst

```diff
@@ -10,7 +10,7 @@ This plugin is located in the ``extensions/plugins/profile.py`` cache folder.
 
 This ``profile.py`` contains a default implementation that does:
 
-- Will try to define ``compiler.runtime_type`` for ``msvc`` compiler if it is not defined, and it will define it to match the ``settings.build_type``.
+- Will try to define ``compiler.runtime_type`` for ``msvc`` and ``msvc`` compilers (in Windows) if it is not defined, and it will define it to match the ``settings.build_type``.
```

Comment:
```suggestion
- Will try to define ``compiler.runtime_type`` for ``msvc`` and ``clang`` compilers (in Windows) if it is not defined, and it will define it to match the ``settings.build_type``.
```